src/ca/wolfram/beta/symath/BaseNode.java

```diff
@@ -0,0 +1,24 @@
+package ca.wolfram.beta.symath;
```

Comment:
the only difference is if people change between constant and variables often it may be annoying to switch all the time. This class keeps everything consistent other than the arguments